jest.config.ts

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  transformIgnorePatterns: ['/node_modules/'],
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/src/$1',
+  },
+};
```

Comment:
We use this file to jest config -> https://github.com/detrash/recy-api/blob/staging/test/jest.shared.ts

Why you need create this aditional config ?

Reply:
You're right, it's really not necessary. I was having trouble running the tests due to the logger module implementation, but I ended up resolving it without needing that and didn’t remove it. I'll delete this file.